docs/using-radicle/identity.md

```diff
@@ -0,0 +1,80 @@
+---
+id: identity
+title: Manage your Radicle identity
+---
+
+Your Radicle identity is your ticket to hosting and collaborating on code in the Radicle network, but it's not a fixed
+entity. You can manage multiple identities, creating/delete them at will, and even edit them to a degree.
+
+Many of the actions below require your personal URN, which you can find using `rad self`.
+
+## View your active identity
+
+Run `rad self` to see details on how you're identified on the Radicle network, including your:
+
+- Active [display name](#edit-your-display-name)
+- Personal URN
+- Peer ID
+- Hashed and full encryption keys
+- File locations of your Git monorepo and encryption keys
+
+## Create a new identity
+
+Add new identities at any time. For example, use different identities for the personal projects you host on Radicle
+versus your professional work for a company or DAO.
+
+```
+rad auth --init
+```
+
+## Switch between identities
+
+If you have multiple identities, switch between them with `rad auth`.
+
+Choose the identity you want active, enter your passphrase, and hit **Enter**.
+
+## Edit your display name
+
+Change your _display name_, which is how you're identified on the Radicle web app (not the same as your Peer ID or
+personal URN!):
+
+```
+rad edit
```

Comment:
Which version of rad is this valid for?

I am getting this:
```
(base) bordumb@bordumbs-MacBook-Pro ~ % rad edit
== Error: rad: command `rad-edit` not found
```

I am on v0.6.x

Reply:
I'm trying to work from the current, which is v0.6.1, where `rad edit` does indeed work for me!

Reply:
Ok, here's my workflow now:

1. Try the above and get that error (I was probably on older version)

2. Reinstall by (a) uninstall `homebrew` and (b) reinstall `homebrew` and then (c) reinstall `radicle-cli`

3. Try `rad edit` but I get the error below

```
(base) bordumb@bordumbs-MacBook-Pro ~ % rad self
Profile b172f291-f9ab-4990-b94e-738796026f50
├── Name           testBordumb                                                                                                   
├── URN            rad:git:hnrkdezzqcy45o487damow6ibwa8kwuodi6bo                                                                 
├── Peer ID        hybg1qak3g5cjgmsxh1oztejsytgwzd9n4z1wqju4eamcfu6w6oxus                                                        
├── Key (hash)     SHA256:A9sS2r0eK5lAj/tRqZ34wJB4T5ZrwpzcWGQy2Mq+318                                                            
├── Key (full)     ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIE0nYVk22JMuz+SheKE2BE1Lj+LV5UcmekYWws/U9B87                              
├── Storage (git)  /Users/bordumb/Library/Application Support/xyz.radicle.radicle-link/b172f291-f9ab-4990-b94e-738796026f50/git  
└── Storage (keys) /Users/bordumb/Library/Application Support/xyz.radicle.radicle-link/b172f291-f9ab-4990-b94e-738796026f50/keys 
(base) bordumb@bordumbs-MacBook-Pro ~ % rad edit
== Edit failed Couldn't get URN from either command line or cwd
```

So maybe this is some kind of bug with previously created identities.
It seems like it knows that I have a previous identity.
However, if I reinstall `radicle-cli` after that identity was created, it won't let me edit it.

Any ideas on whether there is a workaround here? Or if I'm missing something?

Thanks!

Reply:
@joelhans let's change the command to this:

```
rad edit YOUR-PERSONAL-URN
```

Simply running `rad edit` does nothing. But that's not apparent from the current example.

Reply:
I could have sworn that `rad edit` alone worked for me just yesterday, but now I get the same error. I'll amend this to make the URN mandatory.

Reply:
Cool.

I think a nicer UX would be:
- If URN is provided, let user edit that profile
- If URN is not provided, default to the current profile